doc/user/content/sql/drop-materialized-view.md

```diff
@@ -0,0 +1,49 @@
+---
+title: "DROP MATERIALIZED VIEW"
+description: "`DROP MATERIALIZED VIEW` removes a materialized view from your Materialize instances."
+menu:
+  main:
+    parent: commands
+---
+
+`DROP MATERIALIZED VIEW` removes a materialized view from your Materialize instances.
+
+## Conceptual framework
+
+Materialize maintains materialized views after you create them. If you no longer need the
+materialized view, you can remove it.
+
+Because materialized views rely on receiving data from sources, you must drop all materialized views that
+rely on a source before you can [drop the source](../drop-source) itself. You can achieve this easily using **DROP SOURCE...CASCADE**.
```

Comment:
Here, it'd be more helpful to specify when you're expected to need the `CASCADE` option for materialized views (esp. since these are inter-cluster objects now!).

Reply:
Ah, I just copied everything from `drop-view.md`! I assume we should adjust that too?

Reply:
Unfortunately, these DDL pages are also not consistent, so making them so is another thing on my list. You can leave it as is, then!